src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureArea {
```

Comment:
The interface name `FigureArea` is a noun phrase which suggests a data object rather than an interface for behavior. According to the checklist, interface names should not be nouns or noun phrases if they represent an action or behavior. Consider renaming it to something like `AreaCalculable` to better convey the intended behavior.